roles/confluent.test/upgrade_testing/mtls-customcerts-rhel-540-current.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+## Variables
+
+export SCENARIO_NAME=mtls-customcerts-rhel
+export START_BRANCH=5.4.0-post
+export START_UPGRADE_VERSION=5.4
+export END_UPGRADE_VERSION=5.0
+
+echo "Call upgrade script"
+sh ./upgrade.sh
```

Comment:
i think a more common bash practice is to use script arguments instead of env vars, it would look like 
```
sh ./upgrade.sh mtls-customcerts-rhel 5.4.0-post 5.4 5.0
```

makes the upgrade.sh easier to run from the cli

Reply:
Both practices are acceptable in bash scripting, it really depends on what it is your attempting to do.  In this instance, I am calling a script from within a script so environment variables actually make sense here.  If the second script was meant to be run on its own, then I would agree that it should be done by script arguments.